jobs/V3/basic_company_sample.rb

```diff
@@ -12,191 +12,205 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def generate_company
+def job_discovery_generate_company display_name:, headquarters_address:
 	# [START generate_company]
+	# display_name  = "Your company display name"
+	# headquarters_address = "Your company headquarters address"
+
 	require "google/apis/jobs_v3"
 	require "securerandom"
 
 	jobs = Google::Apis::JobsV3
-	company_name = "companyName:"+ SecureRandom.hex;
-	company_generated = jobs::Company.new;
-	company_generated.display_name = "Google";
-	company_generated.headquarters_address= "1600 Amphitheatre Parkway Mountain View, CA 94043";
-	company_generated.external_id = company_name;
-	puts "Company generated: " + company_generated.to_json;
-	return company_generated;
-	# [END generate company]
+	company_name = "companyName: #{display_name}" + SecureRandom.hex
```

Comment:
Since external_id has to be unique, I am using SecureRandom.hex to generate the unique string. Is there a better way to do hat instead of using random hex?